velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -296,6 +298,138 @@ TpchPlan TpchQueryBuilder::getQ6Plan() const {
   return context;
 }
 
+TpchPlan TpchQueryBuilder::getQ11Plan() const {
+  static const std::string kPartsupp_1 = "partsupp";
+  static const std::string kSupplier_1 = "supplier";
+  static const std::string kNation_1 = "nation";
+
+  std::vector<std::string> partsuppColumns_1 = {
+      "ps_availqty", "ps_suppkey", "ps_supplycost"};
+  std::vector<std::string> supplierColumns_1 = {"s_suppkey", "s_nationkey"};
+  std::vector<std::string> nationColumns_1 = {"n_nationkey", "n_name"};
+
+  auto partsuppSelectedRowType_1 = getRowType(kPartsupp_1, partsuppColumns_1);
+  const auto& partsuppFileColumns_1 = getFileColumnNames(kPartsupp_1);
+  auto supplierSelectedRowType_1 = getRowType(kSupplier_1, supplierColumns_1);
+  const auto& supplierFileColumns_1 = getFileColumnNames(kSupplier_1);
+  auto nationSelectedRowType_1 = getRowType(kNation_1, nationColumns_1);
+  const auto& nationFileColumns_1 = getFileColumnNames(kNation_1);
+
+  std::string nationNameFilter = "n_name = 'GERMANY'";
```

Comment:
I do not see this filter used anywhere. Please can you re-check the plan below as it should appear in both the main and sub-query.